internal/runbits/findproject/project.go

```diff
@@ -46,7 +46,7 @@ func FromPath(path string, ns *project.Namespaced) (*project.Project, error) {
 		return nil, &LocalProjectDoesNotExist{locale.WrapInputError(err, "err_project_frompath_notexist", "", path)}
 	}
 
-	if ns != nil && ((ns.Owner != "" && pj.Namespace().Owner != ns.Owner) || pj.Namespace().Project != ns.Project) {
+	if ns != nil && ns.IsValid() && ((ns.Owner != "" && pj.Namespace().Owner != ns.Owner) || pj.Namespace().Project != ns.Project) {
```

Comment:
nit: This could be cleaned up a lot by having IsValid do the nil check, and having a method on ns like namespace1.Matches(namespace2).

Reply:
I've filed this story to follow up:

https://activestatef.atlassian.net/browse/DX-1231